src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { combineReducers, createStoreHook } from 'react-redux';
+import { booksReducer } from './books/books';
+import { categoriesReducer } from './categories/categories';
+
+const rootReducer = combineReducers({
+  books: booksReducer,
+  categories: categoriesReducer,
+});
+
+const store = createStoreHook(rootReducer);
+export default store;
```

Comment:
- [ ] As requested by the previous reviewer, kindly `import { combineReducers, createStore } from 'redux';` instead of `import { combineReducers, createStoreHook } from 'react-redux';`. Kindly use the `createStore` to create a store through the reducer and not `createStoreHook` as that is the requirement for this project.